docs/DevGuide/Notation.md

```diff
@@ -0,0 +1,139 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+# Notation Used by SpECTRE {#spectre_notation_general}
+
+# Current Notation {#spectre_current_notation}
+
+\tableofcontents
+
+## Discontinuous Galerkin Notation {#dg_notation}
+
+- The logical coordinates in the element are
+  \f$\boldsymbol{\xi}=\{\xi,\eta,\zeta\}\f$.
+- The Jacobian matrix is defined as
+  \f{align*}{
+      \mathbf{J}=\frac{\partial \mathbf{x}}{\partial \boldsymbol{\xi}}
+  \f}
+  where \f$\mathbf{x}\f$ are the coordinates mapped to by the coordinate map.
+  The determinant of the Jacobian matrix is denoted by \f$J\f$.
+- We use \f$s\f$ and \f$t\f$ to be grid point indices. That is, \f$U_s\f$ is
+  \f$U\f$ at the grid point \f$s\f$. For tensor product bases we index the
+  individual irreducible topologies by subscripting \f$s\f$. For example,
+  \f$s_1\f$ would be the \f$\xi\f$-dimension in the reference element.
+- We use a nodal basis of Lagrange interpolating polynomials
+  \f{align*}{
+      \ell_{s_i}(\xi)=\prod^{N}_{\substack{t_i=0\\ t_i\ne s_i}}
+      \frac{\xi - \xi_{t_i}}{\xi_{s_i}-\xi_{t_i}},
+  \f}
+  where \f$N\f$ is the order/degree of the basis in each logical direction.
+- A DG scheme with an \f$N^{\mathrm{th}}\f$ order basis is referred to as a
+  \f$P_N\f$ scheme and converges at order \f$\mathcal{O}\left(\Delta
+  x^{N+1}\right)\f$ where \f$\Delta x\f$ is the 1d size of the element.
+- The unit normal vector to one of the surfaces of the element is denoted by
+  \f$n_i\f$.
+- The numerical flux boundary correction term which includes the normal vectors
+  is denoted by \f$G(F^{i,+},n_i^+,u^+,F^{i,-},n_i^-,u^-)\f$ where the \f$+\f$
+  and \f$-\f$ denote the values on the outer and inner side of the mortar,
+  respectively. To give a concrete example, for the Rusanov flux we have
+  \f{align*}{
+      G^{\mathrm{Rusanov}}=
+      \frac{1}{2}\left(F^{i,+}n_i^+-F^{i,-}n_i^-\right)
+      -\frac{C}{2}\left(u^+-u^-\right)
+  \f}
+
+## General Relativity and (Magneto)hydrodynamics{#gr_hydro_notation}
+
+We use the following conventions and notation:
+- In general we work in units where \f$G=c=1\f$.
+- Latin letters at the beginning of the alphabet, \f$a,b,c,\ldots\f$ are
+  spacetime indices ranging from 0 to 3.
+- Latin letters \f$i,j,k,\ldots\f$ are spatial indices ranging from 1 to 3.
+
+The notation for general relativity quantities used is:
+- The Einstein summation convention is used where repeated indices are summed
+  over.
```

Comment:
We also use this convention in situations unrelated to GR.  For example, the Rusanov flux written above.

Reply:
Do you think we should have a `General Notation` section at the beginning that this (and anything else general) could go in?

Reply:
Yes, that sounds good.